tests/test_efadv.py

```diff
@@ -7,10 +7,13 @@
 import errno
 from pyverbs.addr import AHAttr
 from pyverbs.base import PyverbsRDMAError
+from pyverbs.cq import CQ
+import pyverbs.enums as e
 from pyverbs.pd import PD
 import pyverbs.providers.efa.efadv as efa
 from tests.base import PyverbsAPITestCase
 from tests.test_addr import get_global_route
+from tests.test_qp import get_qp_init_attr
```

Comment:
I think get_global_route() and get_qp_init_attr() can be moved to "tests.utils".

Reply:
Ack